pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -175,21 +163,3 @@ def _update_running_loss(self, current_loss: Tensor) -> None:
 
         # reset for next set of accumulated grads
         self.accumulated_loss.reset()
-
-    def get_active_optimizers(self, batch_idx: Optional[int] = None) -> List[Tuple[int, Optimizer]]:
```

Comment:
Is any deprecation needed here? till now it was a public API, since now it is a protected function...

Reply:
I can do that if it is desired. However, at this point we changed already so much in the loops from 1.4 to 1.5 that this one is not going to matter. For example, we moved a lot of public API from one loop to the other in #9191 without deprecations, because it's not really possible to even keep backward compatibility there if we want to move fast.  

@carmocca @justusschock opinions on this?

Reply:
i'm on the yolo side